src/components/category-filter.tsx

```
@@ -0,0 +1,50 @@
+import { Button } from "@/components/ui/button"
+import { BookOpen, Leaf, Cpu, Palette, Heart, Users, LayoutGrid } from "lucide-react"
+
+const categoryIcons = {
+  All: LayoutGrid,
```

ESLint

[error] Inconsistently quoted property 'All' found.

(style/quote-props)